content/en/02-installing-pixie/03-install-guides/02-self-hosted-pixie.md

```diff
@@ -127,11 +127,11 @@ Get Pixie fully managed with [Pixie Community Cloud](/installing-pixie/install-g
 
 ### Authentication using Kratos / Hydra
 
-Self-managed Pixie Cloud only supports one organization and precreates an admin account by default. Login using `admin@default.com` with `admin` as the password.
+Self-managed Pixie Cloud only supports one organization and precreates an admin account by default. The default user is created with `admin@default.com` as the email and with `admin` as the password. If you would like to change the admin user details, modify the email and password in `ADMIN_IDENTITY` in `k8s/cloud/base/ory_auth/kratos/kratos_deployment.yaml`. You will need to redeploy from scratch if you've already deployed with the default admin account, since the admin account has already been provisioned. 
```

Comment:
```suggestion
1. Login to the admin account using `admin@default.com` for the email and `admin` for the password. 

If you would like to change the admin account credentials, modify the email and password values for the `ADMIN_IDENTITY` in `k8s/cloud/base/ory_auth/kratos/kratos_deployment.yaml`. You will need to redeploy from scratch if you've already deployed with the default admin account, since the admin account is automatically provisioned for you.
```